src/domain/orders/new/new-order.tsx

```diff
@@ -54,9 +54,10 @@ const NewOrder = ({ onDismiss }: NewOrderProps) => {
         shipping_methods: [
           {
             option_id: data.shipping_option.value,
-            price: data.custom_shipping_price
-              ? data.custom_shipping_price
-              : undefined,
+            price:
+              typeof data.custom_shipping_price === "number"
```

Comment:
🙏 nice catch. It is important when we work with numbers check the typeof it as 0 is considered as false